instance/models/instance.py

```diff
@@ -398,6 +422,16 @@ def set_fork_name(self, fork_name, commit=True):
         if commit:
             self.save()
 
+    def update_from_pr(self, pr):
+        """
+        Update this instance with settings from the given pull request
+        """
+        truncated_title = truncatewords(pr.title, 4)
+        #pylint: disable=attribute-defined-outside-init
```

Comment:
It's only for the self.name attribute, right? Can this be limited to a single line, to avoid disabling it also for the next one?